src/quiz/quiz.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { QuizService } from "./quiz.service";
+
+@Controller('/api')
```

Comment:
it is better to change /api to /api/quiz so that you won't repeat for every endpoint.